cryptotik/livecoin.py

```diff
@@ -0,0 +1,73 @@
+import requests
+from .common import APIError, headers
+from re import findall
+
+class Livecoin:
+
+    url = 'https://api.livecoin.net'
+    delimiter = "/"
+    headers = headers
+
+    @classmethod
+    def format_pair(cls, pair):
+        """format the pair argument to format understood by remote API."""
+        pair = pair.replace("-", cls.delimiter).upper()
+        return pair
+
+    @classmethod
+    def api(cls, url):
+        '''call api'''
+
+        try:
+            result = requests.get(url, headers=cls.headers, timeout=3)
+            assert result.status_code == 200
+            return result.json()
+        except requests.exceptions.RequestException as e:
+            print("Error!", e)
```

Comment:
Please raise `APIError` here. See other exchange classes for examples (btce or poloniex).